src/main/scala/com/databricks/spark/sql/perf/table.scala

```diff
@@ -47,7 +35,7 @@ abstract class TableForTest(
 
   val name = table.name
 
-  val outputDir = s"$baseDir/parquet/${name}"
+  val outputDir = s"$baseDir/${name}"
```

Comment:
This change is fine.  I'm going to redo all of the data generation at some point anyway now that partitioning is a first class citizen.